components/cluster/command/check.go

```diff
@@ -63,8 +62,6 @@ conflict checks with other clusters`,
 				return cmd.Help()
 			}
 
-			logger.EnableAuditLog()
```

Comment:
Why delete this line? We should record the audit log if user use command `tiup cluster check --apply` because `--apply` will change the system configuration.

Reply:
set in root.go `PersistentPreRunE` instead of every command setting this.